src/main/java/model/EventModel.java

```diff
@@ -72,25 +80,54 @@ public boolean updateEvent(EventBean event) {
 	}
 	
 	public boolean deleteEvent(String id) {
-		return daoFactory.getEventDao().deleteEvent(Integer.parseInt(id));
+		if(checkInteger(id)) {
+			return daoFactory.getEventDao().deleteEvent(Integer.parseInt(id));
+		}
+		else {
+			return false;
+		}
 	}
 	
 	public ArrayList<ParticipantBean> getParticipantList(String id) {
 		ArrayList<ParticipantBean> result = new ArrayList<ParticipantBean>();
-		ResultSet data = daoFactory.getEventDao().getParticipants(Integer.parseInt(id));
-		if(data != null) {
-			try {
-				while(data.next()) {
-					ParticipantBean participant = new ParticipantBean();
-					participant.setEventId(data.getInt("id_event"));
-					participant.setUserId(data.getInt("id_user"));
-					participant.setUsername(data.getString("username"));
-					participant.setStatusId(data.getInt("id_status"));
-					participant.setStatus(data.getString("status"));
-					result.add(participant);
+		if(checkInteger(id)) {
+			ResultSet data = daoFactory.getEventDao().getParticipants(Integer.parseInt(id));
+			if(data != null) {
+				try {
+					while(data.next()) {
+						ParticipantBean participant = new ParticipantBean();
+						participant.setEventId(data.getInt("id_event"));
+						participant.setUserId(data.getInt("id_user"));
+						participant.setUsername(data.getString("username"));
+						participant.setStatusId(data.getInt("id_status"));
+						participant.setStatus(data.getString("status"));
+						result.add(participant);
+					}
+				} catch (SQLException e) {
+					e.printStackTrace();
+				}
+			}
+		}
+		return result;
+	}
+	
+	public boolean resolveParticipant(Map<String, String[]> parameters) {
+		boolean result = false;
+		if(parameters != null && !parameters.isEmpty() && parameters.containsKey("event") && parameters.containsKey("user") && parameters.containsKey("action")) {
+			if(checkInteger(parameters.get("event")[0]) && checkInteger(parameters.get("user")[0]) && checkInteger(parameters.get("action")[0])) {
+				int eventId = Integer.parseInt(parameters.get("event")[0]);
+				int userId = Integer.parseInt(parameters.get("user")[0]);
+				int action = Integer.parseInt(parameters.get("action")[0]);
+				switch (action) {
+					case 1: result = daoFactory.getEventDao().enterEvent(eventId, userId); break;
+					case 2: result = daoFactory.getEventDao().leaveEvent(eventId, userId); break;
+					case 3: result = daoFactory.getEventDao().sendRequest(eventId, userId); break;
+					case 4: result = daoFactory.getEventDao().cancelRequest(eventId, userId); break;
+					case 5: result = daoFactory.getEventDao().approveUser(eventId, userId); break;
+					case 6: result = daoFactory.getEventDao().blockUser(eventId, userId); break;
+					case 7: result = daoFactory.getEventDao().removeUser(eventId, userId); break;
+					default: return result;
```

Comment:
1. some of these cases are unnecessary as they call functions which actually create the "same" sql query.
2. It would be better to use enum for possible actions as it can be reused easily and makes the code easier to understand and maintain it later on.